users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    adress = models.CharField(max_length = 256, blank = True)
```

Comment:
Should be
address = models.CharField(max_length=256, blank=True)
No spaces
aDDress - double d